sass/wroteAboutUs.sass

```diff
@@ -0,0 +1,15 @@
+.wrote-about-us
+    .wrote-about-us__images-container
```

Comment:
use &!! 
see my previous comment carefully

Reply:
sorry for inattentiveness